blog/2023-11-24-data-pipeline-orchestrator/index.mdx

```diff
@@ -0,0 +1,296 @@
+---
+title: Launch Week Day 5 - Windmill for data pipelines
+authors: [guillaumebouv]
+tags: ['Launch week', 'Data Pipeline orchestrator']
+image: ./data_pipelines.png
+slug: launch-week-1/data-pipeline-orchestrator
+description: 'Use Windmill to build, run, and monitor your data pipelines.'
+---
+
+import DocCard from '@site/src/components/DocCard';
+import BarChart from '@site/src/components/BarChart';
+
+![Windmill for data pipelines](./data_pipelines.png 'Windmill for data pipelines')
+
+_Run your ETLs on-prem up to 5x faster using Windmill compared to Spark while simplifying your infra_
+
+Last day of our launch week today (we know it's sad). Before saying goodbye, we want to tell you about features that will help you in your ETL, with restartable flows and S3 integration for data pipelines (this article).
+
+## Running an ETL with Windmill
+
+An ETL is nothing else than a [DAG](https://en.wikipedia.org/wiki/Directed_acyclic_graph) of jobs, each of them reading datasets as input, running computation, and producing new datasets (or updating them).
+
+Windmill can be a great fit for running data processing pipelines:
```

Comment:
Windmill enables building fast, powerful, reliable, and easy to build data pipelines